ProjectK.Business/ExperiencePoints/Experience.cs

```diff
@@ -0,0 +1,61 @@
+﻿using ProjectK.Contracts;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace ProjectK.Business.ExperiencePoints
+{
+    public interface IExperienceEdit
+    {
+        IExperienceEdit ForChar(int charId);
```

Comment:
Method names should be meaningful

Reply:
ForChar == ForCharacter
Its just a shorter version of the same word

Reply:
It's better to use full names and not short-hands. Also `ForCharacter`  still doesn't explain what this method does.